sims.code-workspace

```diff
@@ -110,6 +110,7 @@
       "MSFAA",
       "NSLSC",
       "Overaward",
+      "overawards",
```

Comment:
Overawards? or why did we have all small?

Reply:
Because it was raising it as a warning I fixed adding it to the lib of words.